backend/app/api/review_analysis.py

```
@@ -0,0 +1,813 @@
+        total=len(results),
+    )
+
+
+# ---------------------------------------------------------------------------
+# GET /reviews/trends — 트렌드 데이터
+# ---------------------------------------------------------------------------
+
+@router.get("/trends", response_model=TrendsResponse)
+async def get_trends(
+    period: str = Query("weekly", description="weekly 또는 monthly"),
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)